### ISSUE TITLE
"Korners" bug is back on menus (Qt 6 version)

### ISSUE THREAD

**Issue by crimsonyote** [boehs/Lightly#18]
This occurs for me on Plasma 6.0.1 after installing the Qt6 version from [the associated AUR package](https://aur.archlinux.org/packages/lightly-qt6-git).

Steps to reproduce:
1: Set "Lightly" as application style
2: Ensure "Blur" effect is enabled
3: Change the menu opacity to make it translucent
4: Right click on any app or the desktop, observe the corners of the menu.

![Screenshot_20240313_104850](https://github.com/boehs/Lightly/assets/128944940/b1334f06-16f7-4b7c-b01f-780dff53188b)

SYSTEM INFO:
Intel Core i5-1135G7 (4C 8T)
Intel Iris Xe Graphics (80EU, i915 driver)
16GB RAM @ 3200MHz (2x 8GB)
500GB ADATA Legend 800 SSD

Plasma 6.0.1
Frameworks 6.0.0
Qt 6.6.2
Zen Kernel 6.7.9-zen1-1-zen (64-bit)
Wayland windowing system (This issue also persists on X11)

**Comment by mojtaba82** [comment]
this issue is on wayland. in x11 this issue doesn't exists.

**Comment by crimsonyote** [comment]
Wrong, I checked on both X11 and Wayland. It persists on both. Perhaps it's due to fractional scaling? I have mine set to 125%

**Comment by mojtaba82** [comment]
You right. This issue exists in x11 as well. In addition, in wayland in opaque mode there is also the problem.

**Comment by mojtaba82** [comment]
My global scale is 100% but this issue persists.

**Comment by Bilalharoon** [comment]
on 6.0.4, this bug still perstists

**Comment by Bilalharoon** [comment]
For me its only happening when I right click and only on the corners of the context menu that overlap another window or plasmoid

**Comment by muzammilshafique** [comment]
i also have this weird corners issue i hate it and it's so annoying 
if anyone knows solution let me know 
or if you don't have this issue tell me about where you installed it and whats your configuration
